src/api-docs/source/includes/_wallet_auctions.md

```diff
@@ -2016,7 +2016,7 @@ Create, sign, and send a FINALIZE.
 
 ### HTTP Request
 
-`POST /wallet/:id/finalize`
+`POST /wallet/:id/revoke`
```

Comment:
are you sure this goes here and not L2148? save me a pull and local test?

Reply:
![image](https://media4.giphy.com/media/JGOw5x0LU1h3kKPDm7/giphy.gif?cid=ecf05e47r2gn21hjrmu7cho27x3l5tu21kzer0jfumiaptv3&rid=giphy.gif&ct=g)

Reply:
![image](https://i.pinimg.com/originals/49/0e/58/490e583b40a72a470b4bfe9728e83a5d.gif)